tests/test_localization.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+import unittest
+import sys
+import os
+import shutil
+import build
+import mssqlcli.localized_strings as localized
+
+
+class LocalizationTests(unittest.TestCase):
+
+    def test_product(self):
+        original = self.unicode(localized.goodbye())
+        localized.translation(languages=['ko']).install()
+        translated = self.unicode(localized.goodbye())
+        assert original != translated
+
+    def test_ko_KR(self):
+        domain, ext = os.path.splitext(os.path.basename(__file__))
+        mo_config = {
+            'extraction_target': '{0}.py'.format(os.path.splitext(__file__)[0]),
+            'lang_name': 'ko_KR',
+            'domain': domain,
+            'trans_mappings': {
```

Comment:
Can we checkin the .mo file with the translated 'goodbye!' message? This way, we're testing the same logic the product code is using?

Reply:
discussed offline